src/app.js

```diff
@@ -21,23 +21,25 @@ export default function (options) {
 		}
 	}, options.hooks)
 
-	for (var name in merge(reducers, effects)) {
-		(function (name) {
-			actions[name] = function (data) {
-				hooks.onAction(name, data)
-
-				var effect = effects[name]
-				if (effect) {
-					return effect(model, actions, data, hooks.onError)
-				}
-
-				var update = reducers[name], _model = model
-				render(model = merge(model, update(model, data)), view, node)
-
-				hooks.onUpdate(_model, model, data)
```

Comment:
Well, `_model` got replaced by moving the assignment into the `hooks.onUpdate` call.